### ISSUE TITLE
Help pls 

### ISSUE THREAD

**Issue by Lida1606** [DJj123dj/open-ticket#2]
how to close tickets

**Comment by DJj123dj** [comment]
at the top of the channel you will find a button to close, after that you have to press the ✅

You can also do it with !ticket close

**Comment by Lida1606** [comment]
Well, you have to set that first, but I can't find where

**Comment by DJj123dj** [comment]
But if you create a ticket, then you have at the top of the channel buttons normally

**Comment by DJj123dj** [comment]
unless you delete the top message

**Comment by Lida1606** [comment]
When I'm at home I'll take a screenshot and show how it looks to me I can create one but no channel is created for the ticket I press create but nothing happens except that the bot writes me a private message that I don't  can open further tickets

**Comment by DJj123dj** [comment]
Is good

**Comment by Lida1606** [comment]
![Screenshot_20220327-135530_Discord.jpg](https://user-images.githubusercontent.com/101999385/160280305-f711c621-a977-4797-b250-449c923928cd.jpg)

![Screenshot_20220327-135546_Discord.jpg](https://user-images.githubusercontent.com/101999385/160280316-3f01a5a5-286f-4f37-b34e-233a58118a54.jpg)

![Screenshot_20220327-135611_Discord.jpg](https://user-images.githubusercontent.com/101999385/160280318-3116a0f4-7239-4009-acfb-aa00b586e9f4.jpg)

**Comment by DJj123dj** [comment]
so the problem is that you created a ticket and you are trying to create another ticket but thats imposible, in the future i will be making a feature to allow multiple tickets. 

You can **reset it by deleting the last ticket** you created or if you deleted it you can follow the steps with `!resetdatabase`

**Comment by DJj123dj** [comment]
and the `!ticket close` command must be run inside the ticket itself

**Comment by Lida1606** [comment]
Yes, but I can't make ticketclose in the ticket because none is created

**Comment by DJj123dj** [comment]
thats because you need to reset your database first

**Comment by DJj123dj** [comment]
> so the problem is that you created a ticket and you are trying to create another ticket but thats imposible, in the future i will be making a feature to allow multiple tickets.
> 
> You can **reset it by deleting the last ticket** you created or if you deleted it you can follow the steps with `!resetdatabase`

like this

**Comment by Lida1606** [comment]
ok i'll try if there are problems i'll let you know

**Comment by DJj123dj** [comment]
is good

**Comment by Lida1606** [comment]
ERROR: TypeError [INVALID_TYPE]: Supplied parameter is not a User nor a Role.
ERROR: TypeError [INVALID_TYPE]: Supplied parameter is not a User nor a Role.
ERROR: TypeError: Cannot read properties of undefined (reading 'author')
ERROR: TypeError: Cannot read properties of undefined (reading 'author')
ERROR: TypeError: Cannot read properties of undefined (reading 'author')





I reset the database and it still doesn't work, what can I do?

**Comment by DJj123dj** [comment]
What command did you type when you got this error?
